app/controllers/articles.js

```diff
@@ -1,7 +1,8 @@
 /**
  * Module dependencies.
  */
-var db = require('../../config/sequelize');
+var db = require('../../config/sequelize'),
+    errorHandler = require('./errors.server.controller');
```

Comment:
I am strongly against error codes ever being sent back to client side application. It should only speak HTTP. A 400 or a 500 error can tell the client all it every needs to know. I don't mind the idea of catching errors on the server and logging them for diagnostics later, but don't pass the error on to the client.